.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,4 +5,6 @@ Resolves #
 
 ## How to test it
 
+## Analytics changes
```

Comment:
Do we need this part in the general PR template?

Reply:
We agreed in the [Analytics epic](https://github.com/gnosis/safe-react/issues/3408) to make this part of the feature routine. Perhaps better suited for the feature request template?

Reply:
It won't hurt to be included here